app/views/admin/unsynced_applications/show.html.erb

```diff
@@ -0,0 +1,118 @@
+<% content_for :before_content do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: "Back",
+    href: admin_unsynced_applications_path
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">Unsynced application</h1>
+
+    <%=
+      govuk_summary_list do |sl|
+
+        sl.row do |row|
```

Comment:
This currently duplicates what's in the regular application view. I just replicated it for now to demo, I'll probably combine them into a single partial